manage/upgrading/version_specific_migration/upgrade_to_python3.rst

```diff
@@ -324,24 +343,36 @@ An example for a tox-setup can be found in https://github.com/collective/collect
 Text and Bytes
 ~~~~~~~~~~~~~~
 
-This is by far the biggest issue when porting to Python 3. Read https://portingguide.readthedocs.io/en/latest/strings.html to be prepared.
+This is by far the biggest issue when porting to Python 3.
+Read https://portingguide.readthedocs.io/en/latest/strings.html to be prepared.
 
```

Comment:
"Read the portingguide<LINK>" gives the user a better experience

Reply:
Added label to link in bbfd8f3